### PR TITLE
Modify launch data csv's

### PR DESCRIPTION
**Pull request by harshil21**
Adds:

- Gravity vector to Interest Launch data
- Gravity vector and Quaternions to winter 2023 launch data (december). Also renames that flight csv.

Cherry-picked partially from https://github.com/NCSU-High-Powered-Rocketry-Club/AirbrakesV2/pull/56/commits/32a8084812f8952aad7181ce16eb76db5512f1cf

### CODE REVIEW

**Review by JacksonElia — APPROVED** [review]
I mean I was thinking names like interest_launch and purple_launch but I suppose keeping the stuff about gravity is important. I don't think the dates are important though

File: utils.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def arg_parser() -> argparse.Namespace:
         "--path",
         help="Define the pathname of flight data to use in mock simulation. Interest Launch data is used by default",
         type=Path,
-        default="launch_data/interest_launch_9_28.csv",
+        default="launch_data/interest_launch_9_28_const_grav.csv",
```
[the remark this answers is not in the document]
Comment (JacksonElia) [reply utils.py:86]: bro did not make it simpler to type

[the remark this answers is not in the document]
Comment (harshil21) [reply utils.py:86]: Does your Tab autocomplete not work? Dates are important cause we had two launches, one in dec and nov.

[the remark this answers is not in the document]
Comment (JacksonElia) [reply utils.py:86]: idk if mine works or doesn't but either way we might as well shorten them as much as possible
e.g.
`interest_launch_const_grav.csv`
`purple_launch_nov_something.csv`
`purple_launch_dec_something.csv`

[the remark this answers is not in the document]
Comment (harshil21) [reply utils.py:86]: I'm going to add an accompanying README for this instead, so you can have shorter names but the exact changes and dates will still be visible to everyone in the future.

**Pull request merged**